### ISSUE TITLE
[low vision a11y] Notepad++ is not accessible with ZoomText 11

### ISSUE THREAD

**Issue by bascan** [notepad-plus-plus/notepad-plus-plus#2918]
### Description of the Issue
When I edit a file in Notepad++ and I have ZoomText 11 enabled then the ZoomText magnifier does not follow the text cursor in the Notepad++ editor pane.

### Steps to Reproduce the Issue
1. Download and install the current ZoomText 11 trial version from [zoomtext.com](https://www.zoomtext.com/download-zoomtext-free-trial/)
2. Start ZoomText.
3. Start Notepad++
4. Type a few lines of text.

### Expected Behavior
I expect the magnifier to follow the text cursor so that it remains visible.

### Actual Behavior
The text cursor "leaves" the magnifier at some point and it becomes invisible.

Without the ability to follow the text cursor it is very hard for a low vision user to use a text editor. The magnifier needs to be manually adjusted all the time, using a mouse or keyboard shortcuts.

**Comment by bascan** [comment]
This is a ZoomText issue and it can be solved by doing the following:
While being in the Notepad++ editor and the cursor being visible in ZoomText's view, press Control+Alt+Shift+D. ZoomText will confirm that a new application cursor is detected and will start tracking it.
